builders/get-started/endpoints.md

```diff
@@ -59,3 +76,23 @@ To create a custom OnFinality endpoint, go to [OnFinality](https://onfinality.io
 3. Your custom API endpoint will be generated automatically
 
 ![OnFinality](/images/builders/get-started/endpoints/endpoints-2.png)
+
+### Pocket Network {: #pokt }
+
+[Pocket Network](https://pokt.network/){target=_blank} is a decentralized node service that provides a free personal endpoint to dapps on Moonbeam & Moonriver.  
```

Comment:
I'm sure Alberto probably showed you the docs standards wiki, but in case he didn't you can check it out here: https://purestake.atlassian.net/wiki/spaces/DR/pages/1326383211/Docs+Standards#Text-Formatting

It can be a lot to digest so really just a reference point!